src/main/java/domain/CarDTO.java

```diff
@@ -0,0 +1,10 @@
+package domain;
+
+public class CarDTO {
+    public final  String name;
```

Comment:
```suggestion
    public final String name;
```

사소하지만, 공백이 한 칸 더 들어갔네요!
코드 작성 후에 reformat을 한 번씩 해주면 좋아요 (맥북 기준 Option + Command + L을 함께 누르면 됩니다)